database/Model.js

```diff
@@ -0,0 +1,128 @@
+const Sequelize = require('sequelize');
+const db = require('./index.js');
+
+// Defines items table
+const Items = db.define('items', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER,
+  },
+  name: Sequelize.STRING,
+},
+{
+  timestamps: false
+});
+
+// Defines users table
+const Users = db.define('users', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER,
+  },
+  username: Sequelize.STRING,
+},
+{
+  timestamps: false
+});
+
+// Defines questions table
+const Questions = db.define('questions', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER,
+  },
+  itemId: {
+    type: Sequelize.INTEGER,
+    onDelete: 'CASCADE',
+    references: {
+      model: 'items',
+      key: 'id',
+      as: 'itemId',
+    }
+  },
+  userId: {
+    type: Sequelize.INTEGER,
+    onDelete: 'CASCADE',
+    references: {
+      model: 'users',
+      key: 'id',
+      as: 'userId',
+    }
+  },
+  question: Sequelize.STRING(510),
+  rating: Sequelize.INTEGER,
+  createdAt: Sequelize.STRING,
+},
+{
+  indexes: [
+    {
+      fields: ['itemId'],
+    },
+    {
+      fields: ['userId'],
+    }
+  ],
+  timestamps: false
+});
+
+// Defines answers table
+const Answers = db.define('answers', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
```

Comment:
maybe abstract some of these common denominators into an object to reduce code size